test/workers/tipline_statistics_periodic_check_worker_test.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../test_helper'
+
+class TiplineStatisticsPeriodicCheckWorkerTest < ActiveSupport::TestCase
+  test "should notify Sentry for missing tipline statistics updates in last 24 hours" do
+    t = create_team
+    # Verify both created/updated dates
```

Comment:
@caiosba and @melsawy just out of curiosity: is there any advantage in using travel_to instead of stubbing the time?

something like:

```ruby
  travel_to 3.days.ago do
    @ms = create_monthly_team_statistic team: t
  end

  CheckSentry.expects(:notify).once
  TiplineStatisticsPeriodicCheckWorker.new.perform
```

Reply:
Good point, @vasconsaurus ... I also prefer and use `travel_to`, for two main reasons: it's part of standard Rails, and works with any time library, while `Time.stubs(:now)` is only affecting specifically `Time.now` and it's part of `mocha`... @melsawy , I suggest you make this change.

Reply:
Done